package.json

```diff
@@ -3,14 +3,17 @@
     "private": true,
     "type": "module",
     "scripts": {
-        "dev": "concurrently 'pnpm run watch:content' 'pnpm run dev:docs'",
-        "dev:docs": "pnpm --stream --filter {docs} dev:docs",
-        "watch:content": "chokidar 'packages/components/**/*.mdx' -c 'pnpm run dev:content'",
-        "dev:content": "pnpm --stream --filter {docs} dev:contentlayer",
+        "storybook": "concurrently 'pnpm run storybook:new' 'pnpm run storybook:legacy' 'pnpm run storybook:main'",
+        "storybook:new": "pnpm --stream  --filter {packages/components} storybook:background",
+        "storybook:legacy": "pnpm --stream  --filter {packages/fondue} storybook:background",
+        "storybook:main": "wait-on http://localhost:6006 http://localhost:6007 && pnpm --stream  --filter {storybook-docs} storybook",
         "build:fondue": "pnpm --stream --filter {packages/fondue} build",
-        "build:docs": "pnpm --stream --filter {docs} build",
-        "lint": "pnpm --stream --filter {packages/**} --filter {docs} lint",
-        "typecheck": "pnpm --stream --filter {packages/**} --filter {docs} typecheck",
+        "build:storybook": "pnpm run build:storybook-new && pnpm run build:storybook-legacy && pnpm run build:storybook-global",
+        "build:storybook-new": "pnpm --stream --filter {packages/components} build:storybook ",
+        "build:storybook-legacy": "pnpm --stream --filter {packages/fondue} build:storybook ",
```

Comment:
We don't need to add all these build scripts, we will run the build in CI so we can keep our local development script more lean in here.